frontend/src/components/accounts/__tests__/AccountDetails.test.tsx

```diff
@@ -1,47 +1,10 @@
 import { renderI18nElement } from "../../../libraries/tester";
-import Enzyme, { shallow } from "enzyme";
-import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
-
-Enzyme.configure({ adapter: new Adapter() });
 
 import AccountDetails from "../AccountDetails";
 
 describe("<AccountDetails />", () => {
   beforeEach(() => jest.resetAllMocks());
 
-  it("should load income and outcome transactions count", async () => {
-    const wrapper = shallow(
-      <AccountDetails
-        account={{
-          accountId: "megan.near",
-          createdAtBlockTimestamp: Number(new Date(2019, 1, 1)),
-          createdByTransactionHash: "Genesis",
-          deletedAtBlockTimestamp: null,
-          deletedByTransactionHash: null,
-          stakedBalance: "0",
-          nonStakedBalance: "654345665432345678765",
-          storageUsage: "876",
-          lockupAccountId: "gjturigjgkjnfidsjffjsa.lockup.near",
-        }}
-        currentNearNetwork={{
-          name: "testing",
-          explorerLink: "http://explorer/",
-          aliases: ["alias1"],
-          nearWalletProfilePrefix: "http://wallet/profile",
-        }}
-      />
-    );
-
-    const outTransactionsCount = 19;
-    const inTransactionsCount = 11;
-
-    wrapper.instance().state.outTransactionsCount = outTransactionsCount;
-    wrapper.instance().state.inTransactionsCount = inTransactionsCount;
-    wrapper.update();
-
-    expect(wrapper.instance().state).toBeDefined();
```

Comment:
This test didn't make much sense, as it tested that `state` exist, but I don't see no way how this could be false.
We should probably implement tests with WAMP mocks

Reply:
Agree, I believe this test is not relevant anymore after we started to collect those data from *near-analytics*

Reply:
Just in case - I pointed out that test didn't make sense because it didn't check anything, not because the data was fetched from a different source